python/cuml/tests/test_linear_svm.py

```diff
@@ -41,15 +41,15 @@
 
 def good_enough(myscore: float, refscore: float, training_size: int):
     myerr = 1.0 - myscore
-    referr = 1.0 - refscore
+    refer = 1.0 - refscore
```

Comment:
Please revert the changes to this module. "Referr" is short for "reference error".

Reply:
done